CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing Guidelines
+
+Welcome to Shipwright Build. We are excited about the prospect of you contributing to our project. Your support is more than welcome!
+
+## Getting Started
+
+We have initial documentation on how to start contributing here:
+
+- Learn how to [change shipwright and try out your changes on a local cluster](/docs/development/local_development.md)
+- Our main [documentation](/docs/)
+- Our [Code of Conduct](/code-of-conduct.md)
+
+## Creating new Issues
+
+We recommend to open an issue for the following scenarios:
+
+- Asking for help or questions. (_Use the **discussion** or **help_wanted** label_)
+- Reporting a Bug. (_Use the **bug** label_)
+- Requesting a new Feature. (_Use the **enhancement** label_)
+
+_Note_: Do not hesitate to contact us inmediately in our [Slack channel](https://kubernetes.slack.com/archives/C019ZRGUEJC).
+
+## Writing Pull Requests
+
+For making `noice` in the repository and creating new contributions, here are some recommendations:
+
+- Fork the project
+- Create a new feature branch for your contribution
+- Make your changes
+- Ensure tests are passing
```

Comment:
Running e2e tests is a high bar for a new contributor. Ensuring unit tests pass is IMO the spirit of this recommendation, and something that should be doable on any developer's machine.

```suggestion
- Ensure unit tests are passing by running `make test-unit`
```

Reply:
I am not necessarily agreeing to you assessment @adambkaplan, if I make code changes, I will want to test them. So, I am likely successful in doing `make local`. From there, it is only a small step to run the unit tests because one definitely has a k8s cluster.

Another suggestion @qu1queee: we do not only look for coders. Writers are also welcome. So, I would phrase it like *If you make code changes, ensure tests are passing"

Reply:
+1 @SaschaSchwarze0 - and I forgot that this section is "recommended" actions. Perhaps we link to the `HACK.md` document which has instructions on running the test suites.

Reply:
done